src/compliment/core.clj

```diff
@@ -66,9 +66,10 @@
                                    (all-sources sources)
                                    (all-sources)))
              candidates (mapcat (fn [f] (f prefix nspc ctx)) candidate-fns)
-             sorted-cands (if (= sort-order :by-name)
-                            (sort-by :candidate candidates)
-                            (sort-by :candidate by-length-comparator candidates))
+             sorted-cands (case sort-order 
+                            :by-name (sort-by :candidate candidates)
+                            :by-length (sort-by :candidate by-length-comparator candidates)
+                            candidates)
```

Comment:
If this is unlikely to break clients, I'd love to make this change. Currently I have a keyword completer powered by pathom indexes that does its own sorting as part of the completion... I'd like to have an option (`:sort-order nil`) that prevents meddling with my sort 😂.

Reply:
I don't mind this, but:

1. Please, at least make it a separate commit.
2. I'd rather see an explicit keyword like `:none`.